contracts/MarginCalculator.sol

```diff
@@ -43,16 +64,17 @@ contract MarginCalculator {
      */
     function getExpiredPayoutRate(address _otoken) external view returns (uint256) {
         require(_otoken != address(0), "MarginCalculator: Invalid token address");
+
         OtokenInterface otoken = OtokenInterface(_otoken);
-        require(now > otoken.expiryTimestamp(), "MarginCalculator: Otoken not expired yet");
+        uint256 expiry = otoken.expiryTimestamp();
+
+        require(now > expiry, "MarginCalculator: Otoken not expired yet");
 
         FPI.FixedPointInt memory cashValueInStrike = _getExpiredCashValue(_otoken);
 
```

Comment:
Can we not use overload on `_getExpiredCashValue` and change this to use the new `_getExpiredCashValue`? 
I don't see the old `_getExpiredCashValue` worth existing since it's only used once here.

Reply:
the `_getExpiredCashValue()` is used in `getExpiredPayoutRate()` that only take an Otoken as arg, and does not use a vault, I don't see a way to refactor it using the `VaultDetails` struct.

Reply:
Actually I think I'm wrong, we can use only the new `_getExpiredCashValue` implementation... I'm thinking of a small refactoring in the Controller also that will reduce gas for `_redeem`. Will include the new refactor there.